build/source/driver/multi_driver.f90

```diff
@@ -128,70 +143,78 @@ program multi_driver
+character(len=256)               :: fileout=''                 ! output filename
+! define model control structures
+integer(i4b)                     :: nLayers                    ! total number of layers
+integer(i4b),parameter           :: no=0                       ! .false.
+integer(i4b),parameter           :: yes=1                      ! .true.
+logical(lgt)                     :: computeVegFluxFlag         ! flag to indicate if we are computing fluxes over vegetation (.false. means veg is buried with snow) 
+type(hru_i),allocatable          :: computeVegFlux(:)          ! flag to indicate if we are computing fluxes over vegetation (.false. means veg is buried with snow) 
+type(hru_d),allocatable          :: dt_init(:)                 ! used to initialize the length of the sub-step for each HRU
+type(hru_d),allocatable          :: upArea(:)                  ! area upslope of each HRU 
 ! exfiltration
-real(dp),parameter        :: supersatScale=0.001_dp         ! scaling factor for the logistic function (-)
-real(dp),parameter        :: xMatch = 0.99999_dp            ! point where x-value and function value match (-)
-real(dp),parameter        :: safety = 0.01_dp               ! safety factor to ensure logistic function is less than 1
-real(dp),parameter        :: fSmall = epsilon(xMatch)       ! smallest possible value to test
-real(dp),allocatable      :: upArea(:)                      ! area upslope of each HRU
-! general local variables
-real(dp)                  :: fracHRU                        ! fractional area of a given HRU (-)
-integer(i4b)              :: fileUnit                       ! file unit (output from file_open; a unit not currently used)
-character(LEN=256),allocatable :: dataLines(:)    ! vector of character strings from non-comment lines
-character(LEN=256),allocatable :: chardata(:)     ! vector of character data
-integer(i4b)              :: iWord                          ! loop through words in a string
-logical(lgt)              :: flux_mask(size(flux_meta))     ! mask defining desired flux variables
-real(dp),allocatable      :: zSoilReverseSign(:)            ! height at bottom of each soil layer, negative downwards (m)
-real(dp),dimension(12)    :: greenVegFrac_monthly           ! fraction of green vegetation in each month (0-1)
-real(dp),parameter        :: doubleMissing=-9999._dp        ! missing value
-logical(lgt),parameter    :: overwriteRSMIN=.false.         ! flag to overwrite RSMIN
-real(dp)                  :: notUsed_canopyDepth            ! NOT USED: canopy depth (m)
-real(dp)                  :: notUsed_exposedVAI             ! NOT USED: exposed vegetation area index (m2 m-2)
+real(dp),parameter               :: supersatScale=0.001_dp     ! scaling factor for the logistic function (-)
+real(dp),parameter               :: xMatch = 0.99999_dp        ! point where x-value and function value match (-)
```

Comment:
The parameter here (now deleted) has a different purpose. It is used to identify the x-value when the logistic function equals xMatch.Not addressed since now deleted.